src/scripts/littler.R

```diff
@@ -1,4 +1,4 @@
-ExcludeVars <- c("R_SESSION_TMPDIR","R_HISTFILE","R_LIBS_USER")
+ExcludeVars <- c("R_SESSION_TMPDIR","R_HISTFILE","R_LIBS_USER","R_LIBRARY_DIR","R_LIBS","R_PACKAGE_DIR")
```

Comment:
When run from inside a package install there are more variables present then when run from the command line. Try this:
~~~
$ tar -z -x -f littler_0.3.0.tar.gz 
$ mkdir /tmp/littler-test-install
$ R CMD INSTALL -l /tmp/littler-test-install littler
* installing *source* package ‘littler’ ...
[ ... skipping some lines of the output ... ]
* DONE (littler)
[ellert@ellert R-littler]$ grep littler-test-install littler/src/littler.h
"R_LIBRARY_DIR","/tmp/littler-test-install",
"R_LIBS","/tmp/littler-test-install",
"R_PACKAGE_DIR","/tmp/littler-test-install/littler",
~~~